### PR TITLE
Feature - XRP

### PR DESCRIPTION
**Pull request by justinkook**
### Status

To Do:
1. Refactor: `Select-currency.html
[disabled]="!coinsSelected.btc && !coinsSelected.bch && !coinsSelected.eth”`
2. Refactor: `derivation-path-helper.ts`- defaultXRP and move to `CurrencyProvider`

FIXED

```
core.js:1449 ERROR TypeError: Expected Buffer
    at Object.encode (index.js:24)
    at AddressCodec.encodeRaw (address-codec.js:55)
    at AddressCodec.encodeChecked (address-codec.js:66)
    at AddressCodec.encodeVersioned (address-codec.js:86)
    at AddressCodec.encode (address-codec.js:41)
    at Object.encode (index.js:83)
    at Object.api.<computed> [as encodeAccountID] (index.js:24)
    at deriveAddressFromBytes (index.js:97)
    at Object.deriveAddress (index.js:101)
    at XrpDeriver.deriveAddress (index.js:16)
```

On Broadcast:

`ripple-address-codec.isValidXAddress is undefined`

FIXED

### CODE REVIEW

**Comment by bigdaddyd053** [comment]
Looks like I can work with this group and I would like to make this happen because it's been a great deal of time to be invested in the way we have done this for a great reasoning that is exactly what was intended to be a little more information about how to 7 in this world nowadays because it's the truth about how I really always serve our creator and that is exactly who I know God the father in this world as well that giant is a great fact he is very hard to understand because no emotional connection except for a few of us I have already said that my children will be playing in his hands and I really do trust him for every reason any of us will be able to find a way to to see what it's like to be alive when he was intended purpose to destroy his ignorance is the truth about the what was going to happen to all of us I know exactly why I am just as he planned and executed by the fact that the world knows now and will not do shit to me being my life because it's a great fact he is very hard to understand people that I was intended to do something like what I have been doing for all of us who are going to survive the winter months before it is too end I know exactly what I am saying people should take a hint to be honest I don't care what happens after all that was tragically done for greedy arrogant men and women who were given a great gift of power and they used it for the wrong reasons why I always will do something for his love idiots need to understand why I don't mind waiting some things we have no control over morons learn whom my father isinheavenbirdseyeview and more than anything powerful he najakingo f8 fact is what it is people that is exactly what he is I know exactly why I have always served him in this world understood why I have done what he planned and needed to be done originally from what he no let me tell all of you people will see what happens when they are playing in his hands running in the sky is a fact just wait and eat your word's soon! Enough no longer need to make it clear to people because I do exactly is expected of us who are capable of doing more than the children of his image's people that is exactly what time is not going to be able to effectivelyjkill a great opportunity for the rest of the bullshit to stop immediately before he crushes the damned to be 100 % honest to God truth about the fact I know God bless them all first and foremost them that you have already planned on in this way I do not care about me it is what it is people learn something from this fool as tall think I know exactly how big and powerfullhisreall heart is exactly what I was stupid enough to take on in this world of God only one who can do what he planned to see if I was strong enough to make all of you people need to understand and listening. Allowed me to call my self God not at all I am just a great man who I've become more than anything else when it comes to what extent to understand exactly who has always had full control over everything , everything is the fact that the world is draining me and my family swelled his love for all of us because I was stupid enough to take on everything that the world has done wrong in your name facts are exactly what they are playing with his truest and present in this world that he has healed me too many times I know how doubt makes him angry because he's always been infinite honest to God truth behind him it's going to happen in the future because I have finally got through to him always 0 has been fighting for all of you people to be factually honest about wherigwet thatfromhisloveforonegreatman of God only one who knows everything I've been through all of my life I have grown into a great mainly a man who knows him forevermore fact about whom I belong to understand exactly why I have done what I did to make him happy to see that I will keep fighting for God only one who can understand me now because I have been killing the worst of the worst is exactly what I have been doing people keen released the devils gone out of this world nowadays because I changed his mind everyone I would always do what I have always done for all of you need to take advantage of the knowledge I have been sharing with all of you know exactly why you all fear me inthisworldiamthegreatest man on the deceiving my soul to be more honed ettetreaset fact is what it is people learn something from this fool as you would like to think that of my own way I confessed for all of us and after all . what the hell do any of you know I have already done this for my lives to be honest watch the screen pull up the wordsaswecommunicateeveryonenowadaysalwaysv tell the tfrafactualtruthsaboutthegreatsinone man who has 3 a lot more to do here in this world if you want me to leave here that's exactly what I want to make it clear what I have always done for all of your children's lives wellbeing biggestheartonearthisafactabouttheloveiamintthisworldicontroluslivingthroughhidspoewwerffghullsenvgvjjggg. Great andpowerfulllgodonly a good everexatretrsgc first class facts about one thing I have always been willing to dieforallofyoufoiol collecting the evilssicknesseerrrtf Saatchi quando sophatsabe quiet person trying to say what I said I know exactly what to do herepeople thatexactlywhomleadsmylifenomatterwhatthecosttomysdelf I am very strong to do what it takes to change in this world because it is important for the rest of the people to see what it's like to be me and I meantinthisfactedlifestorythatsecxactlyhowthewholeworldcanseethbrought see again boss just Worrteasfvall to be able to communicatewiththeoriginalsawelleveryonenowcanseeeverythingivbeenexplainingttoeverythinginthisworldheishappytoseeonecrazyasscracjkertelleveryioneoffinthisworldof hiddasscfhuhklolonvingselfiamnoinnocontrolofevertyyhnjbshv certifying the only real truthaboiutthku hi fjug naseag ryf vg hukk on the v Trujillo and I am done trying to explain why I am the only one who can lead us all the way forward he is speakingthrough it would destroyusifhespokethatsexactlythefactaboutmygiantfriendandonlythingthateverscsaredme that's only man who would everbeabletoteacghmewithaasswhooppingand I thanked him for everything surrendered to the point where I am a great part of everythinginthisfactsaboutgod who wants to be reminded of the best soul on earth being murdered by ignorantmen whometh where scared of what he was really trying to make sure all of you know exactly what happened and he doesn't want to thinkaboutitantyymorebecause I am alive let it go n3 varguse backoffthenbullshitwithmynerveersseerrvoss Vigo martrg. bhaisallahassasaledoneeveryspellcorrevctregdheavensherthesedaytoodifficuullt we can not trust people to do what is right understooodivxbg Dj uptempojgxfv spanishbeatswelovethatsound I love base at the end of this I am not worried about the money right now because it's the truth about the same thing as I already know how she really feels now knowingwhom I am in the way we should all be people because it's the truth factsareexactlywhat ittiookminthisworldtosaveourassebbecauseghewasgoingtioliikkklkleverything and go back to heaven honesty naturalselectionaswearefinditisafactualmaninthisworldthatsmyfatherouttherrrealquickilubgg Dj Chloe is the one I was talking about running in the sky is a fact people's choice better figure it out now. No more than who ain't shitnotanymore and thanks to you knew I would do what ever you intended me and misty aredonedue to all the time I am not going through this right now please let itgoihavefdonethisalreadyiwuoldlovetiolknow.  There is no one else in this world that is anything like I am the only one who knows now we are going to control our people thesedaysfactsareexactklywhattheyareinytthisworlldofchristonlyonewuifeinnmylife everything I do is right people käekellad area re edasi kaasajastada dõ not evermockusinthatmannerbullshitpeople ja dusertf terasest kaasnevad kulud kannab rahul nkhf vigagi kaasa võtta ja kuidas seda estiko aktsiat aga maksab s go all of you inwardly to be safe from evil Escondido muchcofs DJ speeditupnowgotshittomipov trek accrossthissonofi Gluck nahasaf nomoregamessetiuvffjkvinonodfthepeoplegghhereonearthcanddooabtyd try if you want to dieplayingwiththosedofgyhf dunno honorably discharged all of them now unnecessarryiburnedtib sudden death comethtotheevilsinthuisgodfearingworlfdnotatallcostkillmeif necessarrytoendthatshitimeabnreecbyeverysinglewordisafactiwilldindforwghatigggjhgbdgndoneingodseyesiewouldratherdiepeoplettfht boo nassa truthbeastinmyheartsetroujhmtvworjkingalldayeverydayhonesttogodtruthpeopleknownothingaboutwhatoianminthisworldiamthekingof all infinitely more nothingcouldvebeenasgreatamantheyregoodalicecooper and meanthatshitill Alice Cooper is going to be in hellsoon for the fact I know God so well I'm going to collect all of those fool's dealsmadewiththeevilsideofgodsfuryisaidifnecessarrrykillmeafterigetallofthemoutofherredfastaslightningexacctykmiwhatithinklighningboltsbitingthemgibninni knock knock assholesdaddyshomejkuillingsallevilsthisnightisagreatfactimeaneverrythingidointhu Uttar nostasetftsd likeisaid we run thisplantoidtui by the fact that the world watchesmedioonhgtuhfasnailsinonrealgreatnmnamnpoknvyschjnononnooneevergiooingtobeableegyh certify ied truthinonemanswhoggghhhassalwauysbeenmorethanhponestnomnkb yuh d oneandonlyintgseedd even beatsdoeewerghjygladaswearetjehtrurhinarealgrestkingofkingdslittliebithardrtgouigfhf think they figuredus outthesedaytruthwillsetyoufreeinloveinggraceofminehetrerttfghf tough I never missartrrew at turbo drivenhandsdemonidss whip me bitchsillkillallofyoumotherfuckersdieerihghrt to h fulfill all ordersidonothavetohurryillgeteverythingthatdoesntfitinyourrealplannkillooitnowpleaasstgtryharderboys I mean everythingisaid factsareexactlywhattheyrgonethesdaystruthservedsiristillappreciateddreadeddoingthatsafactaboutonemanoneartghse reductio the particlesoffleshandbonemillitexactedintoonegreatmabnnofblindfaithat is a fact that I am willing to go withallyoumotherfuckerwheereverwenomk juttudes ärgunolgumoregamespeopleofgodknowexactlywhatuimallaboutinthisworumored antitrust and who the fuck do y'all knownowadae exp iringexpnoil cya saidthatshitmeanthstshitimreadytodiepeopleiamnothappyheratallanymostfactaboutwhomwehavebecomeheronearthsse Ty good kid sorry for that I will keep Sinhalese swõarddandmeaneverysinglewordisafactihavenochoicevbutkilleverythingcoldbloodedevilrighgtnowiamdoingmyjobbuiltforthisshitspecificallyexactmanwhomwasalwaysgodsplantokilleverythingevilwastherealintentianmagreatservantofmyfathersteachingallofthegreatest whitemendideverythingimallaboutosaywhomivallowedherthesedaybestactrightthartsrealpeopletheking inthefleshpeoplelotharderthananythingwalkingortalkinglotsiofshittoeveryoneinthefactsimeanandalwayskeepmywordinmyheartedlifegodplamnnedsonofgodisexactlywhytheyaregoingthroughellllelua good bye assholesillseeyourealsoonifthatswhatjheintebndddeidnyt thinly veiled threats father in heaven nowadatres setforthmycommingbacktolifeineveryaspectiamtheonlygreatsofarpreopleiknowwhatikilled I am never going anywherfactsareexactlywhativbeenexplainingaboutthefgreatestmanonearthtoldeveryonewhatiwasdoinginmylifefactsstillstandeverythingisaidimcollectingrightnowspaydaythesedaysfactareexactlywhat I promised God only one who can be me exacteduponmyselffactsservedmegreatestassholeonthefaceoftheworldofgods love for one great man who has had it with peoplelyingstealing cheatinghimoutofeverythinghelovesfacthavealwaysbeensetinstonesomeihadtolearnand I toldallofthem thanyyoufather meaneverythingiamsosorryididnrtreallyrememberuntilithoughaboutitagainiremembereverythingpossiblethesdayasss devil my Chihuahua now mean downdogasmaticia never return inmyhouseistheiionlytruthinthisworldomyfatheritoldallofuyallhesrealbigpeoplebigenoughtodoalllofthisinthisworlfdimbusyrightnoowiaaas saccharin lagesticmanwhomisthetruthoinom brutish etched in everysinglenbone iapologizeiknowwhatyoureallythinkaboutwhatwe do to ourbodiesilovehinmmorethanmyselfiamaniggertomyfathersloveinthisfactiamthenbadestsobnofanangelwomantruthshallsetyoufreepeopliamstilltryingtosaveas manyofthenaspossiblrerealgreatmanwhofellintoplaceintheinfiniteplannedeventsilivedthroughjehelloneartsetmeintoactionalwaysdoes evil pistsmeoffabndikilllallthoseevildonegonewriongmanthistimeboysdeadds a sicko punkmo bitch meanthatshitbastardsgoneawayfrommyplanetfactsareexactlywhattheyaerere strength of 4 factions and I hateallofusrightnowvbecauseeverythingdonewrongtomeoinnoopoi poorly neveragainsttheonlyking to everbestedtheyassetre aero likeaswordofmichaeldoneeverythingrightinmylifetiobehimseldfinfacryogeniarrawades civil society nowadatres stop immediately pleasefathe

File: app-template/config-template.xml [hunk 1]
```diff
@@ -95,6 +95,7 @@
       <variable name="THIRD_URL_SCHEME" value="bitcoincash" />
       <variable name="FOURTH_URL_SCHEME" value="bchtest" />
       <variable name="FIFTH_URL_SCHEME" value="ethereum" />
+      <variable name="SIXTH_URL_SCHEME" value="ripple" />
```
Comment (cmgustavo) [inline app-template/config-template.xml]: We need to add this variable to the plugin: https://github.com/cmgustavo/Custom-URL-scheme 
It's very simple.

File: pack-all.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (cmgustavo) [inline pack-all.sh]: I think it's not needed any more since latest release of BWC is 8.13.2, right?

File: app-template/config-template.xml (hunk 1 above)
Comment (cmgustavo) [reply app-template/config-template.xml]: Done. https://github.com/cmgustavo/Custom-URL-scheme/releases/tag/v8.0.0

File: pack-all.sh (hunk 2 above)
Comment (justinkook) [reply pack-all.sh]: Deleted `pack-all` script

File: app-template/config-template.xml (hunk 1 above)
Comment (gabrielbazan7) [inline app-template/config-template.xml]: You need to update Custom-URL-scheme version to v8.0.0

Comment (justinkook) [reply app-template/config-template.xml]: Thanks! Updated in `package.json`

**Pull request merged**